lib/ignored-paths.js

```diff
@@ -37,20 +36,17 @@ var DEFAULT_OPTIONS = {
 // Helpers
 //------------------------------------------------------------------------------
 
-var ignoreFileFinder;
 
 /**
- * Find an ignore file in the current directory or a parent directory.
+ * Find an ignore file in the current directory.
  * @param {stirng} cwd Current working directory
  * @returns {string} Path of ignore file or an empty string.
  */
 function findIgnoreFile(cwd) {
     cwd = cwd || DEFAULT_OPTIONS.cwd;
-    if (!ignoreFileFinder) {
-        ignoreFileFinder = new FileFinder(ESLINT_IGNORE_FILENAME, cwd);
-    }
 
-    return ignoreFileFinder.findInDirectoryOrParents(cwd);
+    var ignoreFilePath = path.resolve(cwd, ESLINT_IGNORE_FILENAME);
+    return fs.existsSync(ignoreFilePath) ? ignoreFilePath : "";
```

Comment:
We use existsSync in a few more places (cli.js, config-file.js, Makefile.js and tests/lib/cli-engine.js). We should remove if everywhere, probably as a separate PR.